### PR TITLE
Added published study count by GBD region visualization

### PR DESCRIPTION
**Pull request by SeanKennyNF**
Added published study count by GBD region visualization

image
image

### CODE REVIEW

File: app/pathogen/arbovirus/dashboard/page.tsx [hunk 1]
```diff
@@ -14,7 +14,7 @@ export default function ArbovirusDashboardPage() {
       filtersComponent={ArbovirusFilters}
```
Comment (HimanshuRanka) [inline app/pathogen/arbovirus/dashboard/page.tsx:14]: +1 on this change, looks super clean

File: app/pathogen/arbovirus/visualizations/page.tsx [hunk 2]
```diff
@@ -1,45 +1,16 @@
 "use client"
```
Comment (HimanshuRanka) [inline app/pathogen/arbovirus/visualizations/page.tsx:1]: Only note here is that it probably makes sense to merge the visualization and (visualization) directories. I understand why they would be seperate technically but they are never used without each other and hence lets centralize it - and avoid confusion going forward

**Review by HimanshuRanka — APPROVED** [review]
Looks great to me! This is amazing work sean thank you!

File: app/pathogen/arbovirus/visualizations/page.tsx (hunk 2 above)
Comment (SeanKennyNF) [reply app/pathogen/arbovirus/visualizations/page.tsx:1]: Let me see what we can do once there's less open PRs hahaha :)

**Pull request merged**